src/config/logic/LinkConfig.ts

```diff
@@ -30,4 +30,6 @@ export var LinkConfig: {
 		},
 		update: (id: string) => updateGeneralizationLink(id)
 	}
-};
\ No newline at end of file
+};
+
+Object.freeze(LinkConfig);
```

Comment:
This freezes only the first level of the object AFAIK. I think it would be better to use `as const` instead (https://blog.logrocket.com/const-assertions-are-the-killer-new-typescript-feature-b73451f35802/) or use at least the ReadOnly type for extra TypeScript safety.